ui/app/utils/default-job-templates.js

```diff
@@ -22,6 +23,19 @@ export default [
       },
     ],
   },
+  {
+    id: 'nomad/job-templates/default/actions',
+    keyValues: [
+      {
+        key: 'template',
+        value: actions,
+      },
+      {
+        key: 'description',
+        value: 'A raw exec job that shows off Nomad Actions',
```

Comment:
Not a copy writer, but I think this teaser text can be improved to describe what actions are and what they do.

Maybe

> Nomad Actions let job authors describe commands that can be run in one click from the UI or one command from the CLI. This example job has actions for getting job info, alloc info, and the Nomad changelog.

Reply:
This is a big improvement, thanks — I rushed this description.